### ISSUE TITLE
Missing Thai_graphclust_exclusive_model4_heavy 

### ISSUE THREAD

**Issue by FrankYFTang** [unicode-org/lstm_word_segmentation#9]
In ICU4X, we have a Thai_graphclust_exclusive_model4_heavy model in  https://github.com/unicode-org/icu4x/blob/master/experimental/segmenter_lstm/tests/testdata/Thai_graphclust_exclusive_model4_heavy

but this model is not checked in under https://github.com/unicode-org/lstm_word_segmentation/tree/master/Models

**Comment by SahandFarhoodi** [comment]
You are right, this is an error. By looking at `weights.json` file, I can tell that name of the model in ICU4X should be `Thai_graphclust_model4_heavy` and we shouldn't have `exclusive` in the name, because I have grapheme clusters that are not Thai-specific in the dictionary there, such as space.
